cockpitlook.cpp

```diff
@@ -1364,7 +1365,10 @@ int CockpitLookHook(int* params)
 				if (g_bInHyperspace)
 					g_headPos = g_prevHeadingMatrix * g_headPos;

```

Comment:
We need to check what happens with the head tracking while in hyperspace.

Reply:
From what I tested it works